modules/phase_field/include/postprocessors/FeatureFloodCount.h

```diff
@@ -460,7 +460,7 @@ class FeatureFloodCount : public GeneralPostprocessor,
                    unsigned int var_num = invalid_id);
 
   /**
-   * This routine is called on the master rank only and stitches together the partial
+   * This routine is called on the primary rank only and stitches together the partial
```

Comment:
not quite the same context at the contact master/slave issue, but I'd be fine with this change